apps/studio/components/layouts/AuthLayout/AuthLayout.tsx

```diff
@@ -22,14 +22,12 @@ const AuthProductMenu = () => {
     authenticationEmails,
     authenticationMultiFactor,
     authenticationAttackProtection,
-    authenticationAdvanced,
```

Comment:
(i can look into these - just leaving notes for reference) enabled features needs to be updated